lib/foreman_rh_cloud/engine.rb

```diff
@@ -38,7 +38,12 @@ class Engine < ::Rails::Engine
 
         # Adding a sub menu after hosts menu
         sub_menu :top_menu, :foreman_rh_cloud, :caption => N_('RH Cloud'), :icon => 'fa fa-cloud-upload' do
-          menu :top_menu, :level1, :caption => N_('Inventory Upload'), :url_hash => { controller: :'foreman_rh_cloud/react', :action => :inventory_upload}
+          menu :top_menu, :inventory_upload, :caption => N_('Inventory Upload'), :url_hash => { controller: :'foreman_rh_cloud/react', :action => :inventory_upload }
+          menu :top_menu, :insights_hits_import, :caption => N_('Manage insights hits import'), :url_hash => { controller: :'foreman_rh_cloud/react', :action => :insights_cloud }
```

Comment:
what do you think about renaming to "Insights sync"?

Reply:
I don't want to use ambiguous word "sync" - this page should also contain all settings related to remediations e.t.c.

What about "Insights data"?

Reply:
> What about "Insights data"?

works for me, maybe even just "Insights" could fit than